### PR TITLE
AMB-1331 update readme files

### PR DESCRIPTION
**Pull request by artronics**
## Summary
* Routine Change
* :exclamation: Breaking Change
* :robot: Operational or Infrastructure Change
* :sparkles: New Feature
* :warning: Potential issues that might be caused by this change

Add any other relevant notes or explanations here. **Remove this line if you have nothing to add.**

## Reviews Required
* [x] Dev
* [ ] Test
* [ ] Tech Author
* [ ] Product Owner

## Review Checklist
:information_source: This section is to be filled in by the **reviewer**.

* [ ] I have reviewed the changes in this PR and they fill all or part of the acceptance criteria of the ticket, and the code is in a mergeable state.
* [ ] If there were infrastructure, operational, or build changes, I have made sure there is sufficient evidence that the changes will work.
* [ ] I have ensured the changelog has been updated by the submitter, if necessary.

### CODE REVIEW

**Comment by LKFULLER1** [comment]
when running 'make serve', the CLI displays http://localhost:5000! but when trying to follow this link, it includes the !, meaning that it doesn't work. Perhaps this exclamation mark should just be removed, to avoid this happening

**Comment by LKFULLER1** [comment]
feedback2
- ^ in the tests README.
- ‘Test flow to test new develops’ doesn’t make sense - is it ‘developers’ or ‘developments’?
- ‘which is inside the AWS is responsible…’ tripped me up, needs a comma after AWS
- ‘The code for [the] container’ ?

**Comment by LKFULLER1** [comment]
feedback3
- ^ still tests README.
- Unsure about the identifier at stage 4: will they all be NHS0001? Are there rules somewhere on what patterns this identifier have to follow or could i literally write anything?
- Also, unsure what ‘part 2’ is referring to at the bottom, as there is no part 2 anywhere in the document

**Comment by LKFULLER1** [comment]
feedback4
- ^ readme-to-test
- /test should be /tests

File: tests/README.md [hunk 1]
```diff
@@ -18,35 +18,27 @@ Make install will install all the requirements in your local machine that you ca
...
 
 ## Set up to run tests
 
 Install get_token
 -----------------------
-The APIGEE_TOKEN env variable has a a dependency of an utility from Apigee that must be installed.
-
-To install acurl and get_token:
+You need an Apigee access token to run tests. `APIGEE_TOKEN` environment variable must contain this access token. In order to
+get apigee access token you need `acurl` and a utility script called `get_toke`. To install acurl and get_token:
```
Comment (LKFULLER1) [inline tests/README.md:39]: on line 39 it says you need a utility script called 'get_toke' but I imagine this is meant to be 'get_token'

**Review by ChewieCB — APPROVED** [review]

**Pull request merged**